e2e/ChannelApeClient.spec.ts

```diff
@@ -13,7 +13,10 @@ describe('ChannelApe Client', () => {
 
     const channelApeClient = new ChannelApeClient({
       sessionId,
-      logLevel: LogLevel.OFF
+      logLevel: LogLevel.OFF,
+      jitterDelayMsMaximum: 6000,
+      jitterDelayMsMinimum: 3000,
+      maximumRequestRetryTimeout: 20000
```

Comment:
@csimko  I think there is something wrong with the new retry logic. I changed the Production Deployment Test Business subscription to only allow 1 API call per second. When I run the e2e suite locally the whole suite fails way quicker than 20 seconds (which is what mocha opts and max timeout is set to).